pkg/splunk/client/awss3client.go

```diff
@@ -64,20 +70,43 @@ func InitAWSClientWrapper(region, accessKeyID, secretAccessKey string) interface
 func InitAWSClientSession(region, accessKeyID, secretAccessKey string) SplunkAWSS3Client {
 	scopedLog := log.WithName("InitAWSClientSession")
 
+	// Enforcing minimum version TLS1.2
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{
+			MinVersion: tls.VersionTLS12,
+		},
+	}
+	tr.ForceAttemptHTTP2 = true
+	httpClient := http.Client{Transport: tr}
+	disableSSL := false
+
 	sess, err := session.NewSession(&aws.Config{
 		Region: aws.String(region),
 		Credentials: credentials.NewStaticCredentials(
 			accessKeyID,     // id
 			secretAccessKey, // secret
 			""),
 		MaxRetries: aws.Int(3),
+		HTTPClient: &httpClient,
+		DisableSSL: &disableSSL,
```

Comment:
Do we need this? I think by default it is already false.

Reply:
Good point.  Just checked and it should default to `false`, from https://github.com/aws/aws-sdk-go/blob/v1.39.4/aws/config.go#L33-L267 :
```
    // Set this to `true` to disable SSL when sending requests. Defaults
    // to `false`.
    DisableSSL *bool
```
I'll remove this and leave the version check.